### PR TITLE
solution

### PR DESCRIPTION
**Pull request by tiserett**

### CODE REVIEW

**Review by Kulchevych — APPROVED** [review]
Good job

File: src/arrayMethodReduce.js [hunk 1]
```diff
@@ -5,7 +5,19 @@
...
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let previous = initialValue;
+    let startIndex = 0;
+
+    if (arguments.length < 2) {
+      previous = this[0];
+      startIndex = 1;
+    }
```
Comment (dellachaise) [inline src/arrayMethodReduce.js]: You could use ternary operator and rewrite this code to only 2 lines.

**Review by dellachaise — APPROVED** [review]
Well done 👍 

Also you could make your code shorter 👇

File: src/arrayMethodReduce.js (hunk 1 above)
Comment (tiserett) [reply src/arrayMethodReduce.js]: ty, havent mentioned this yet, fixed it! ;)

**Pull request closed without merging**